apps/sms/test/marionette/mocks/mock_navigator_moz_mobile_message.js

```diff
@@ -109,50 +132,49 @@ Services.obs.addObserver(function(document) {
         onerror: null
       }, window));
 
-      window.setTimeout(function() {
+      parameters.type = 'mms';
+
+      createMessage(parameters, options).then(function(message) {
         if (typeof request.onsuccess === 'function') {
           request.onsuccess.call(request, Cu.cloneInto({
             target: { result: null }
           }, window));
         }
 
-        parameters.type = 'mms';
-
         callEventHandlers('sending', Cu.waiveXrays(Cu.cloneInto({
-          message: createMessage(parameters, options)
+          message: message
         }, window)));
-      }, delayMs);
+      });
 
       return request;
     },
 
     send: function(recipients, content, options) {
-      var requests = Cu.waiveXrays(Cu.cloneInto(recipients.map(() => ({
-        onsuccess: null,
-        onerror: null
-      })), window));
+      var requests = recipients.map(function() {
```

Comment:
But we don't use it currently, I left it for the developer who will migrate to then in the code so that mock contains code that is actually used :) Do you think it's better to do it in this patch to support future migration to ```send().then```?